src/screens/home/options/OptionsScreen.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
Lets rename all files to use `kebab-case` notation, in this case: `options-screen.js`